app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_account_update_params, only: [:update]
+
+  def new
+    super
+  end
+
+  def create
+    p "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@", params
+    super
+  end
+
+  def edit
+    super
+  end
+
+  def update
+    super
+  end
+
+  def destroy
+    super
+  end
+
+  def cancel
+    super
+  end
+
+  protected
+
+  def configure_sign_up_params
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+  end
+
+  def configure_account_update_params
+    devise_parameter_sanitizer.permit(:account_update, keys: [:attribute])
```

Comment:
```suggestion
    devise_parameter_sanitizer.permit(:account_update, keys: [:name])
```
podemos add na view /edit pra editar o user